CHANGELOG.rst

```diff
@@ -0,0 +1,15 @@
+================================
+Community.Proxysql Release Notes
+================================
+
+.. contents:: Topics
+
+
+v0.1.0
```

Comment:
IMO this should be 0.2.0, as 0.1.0 has already been released without a changelog. (Galaxy releases are immutable.)

Reply:
Ahh that's useful to know - I think then I'll remove the release, and recreate this as a fragment to release 0.2.0 later along with some more changes.  I'd just included it here retrospectively now to act as a baseline given 0.1.0 was already released, but it sounds like there's no need.

Reply:
Well, that's my personal opinion, you can still ignore it :) We also did the same f.ex. for community.general, the first changelog entry is for 0.2.0, even though there were some 0.1.x releases before (they were essentially for bootstrapping, not for actual use - which is simliar to here).

In fact, there are some collections with 1.0.0 releases that didn't have a changelog so far and which hopefully get one in a 1.x.y release. No idea what they will do about this situation though, if they start with 1.x.y or add "earlier" entries (like you did in this PR).

Reply:
not at all, it's definitely great feedback - keep it coming! 🙂 again I really appreciate the guidance you've given 👍